consensus_pbft.go

```diff
@@ -0,0 +1,785 @@
+package pbft
```

Comment:
Revert name of the file either to `consensus.go` or some other name if you can come up with some more appropriate. :slightly_smiling_face:

Reply:
Well, this file contains the pbft implementation of consensus. Do you think this is not the right name for the file?

Reply:
I was just thinking that `consensus.go` was already simple and good enough. Thought that you've renamed it this way just because you have reserved `consensus.go` for that file which you've now renamed to `consensus_cfg.go`.
I don't have problem to leave it like this if you find it better reflects the purpose. Tbh, it bothered me more naming that other file `consensus` than this naming `consensus_pbft`. 😄